CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ endif ()
 
 option(BUILD_TESTS "Build oeedger8r virtual environment tests" ON)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Why do we need to upgrade version here? Would 11 break something?

Reply:
The library I'm using to create directory (filesystem) is not on C++11

Reply:
Hmm, it seems too disruptive to require C++17 now. Is the C++17 tool chain installed by some script (like OE has install-dependency.ps1) from this repo? Or user has to install on their own

Reply:
Does it disrupt any services that you know of? As far as I've seen only OE uses this library and we can set it to compile independently with C++17 on OE

Reply:
Also yes clang-11 can compile C++17